Machine Learning/Hands-on Implementation using sklearn/Linear Regression/Linear Regression .ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Could you add the credits for the dataset, so the author of this dataset could be appropriately credited?</p><p> </p><p>Could you also add this just under the Heading, so it could become easier for others?</p><p> </p><pre class="ql-syntax" spellcheck="false">_Notebook orignially contributed by: [Aditya-1500](https://github.com/Aditya-1500)_


&lt;table class="notebook-buttons" align="left"&gt;
&nbsp; &lt;td&gt;
&nbsp; &nbsp; &lt;a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;
&nbsp; &lt;/td&gt;
&nbsp; &lt;td&gt;
&nbsp; &nbsp; &lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Linear%20Regression/Linear%20Regression%20.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;
&nbsp; &lt;/td&gt;
&lt;/table&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
As I told in previous reply also, I don't have the details of the author of this dataset. Rest regarding the head, I'll add this.

Reply:
Great, you could now maybe add the suggestions in [this comment](https://github.com/Rishit-dagli/Python3-by-practice/pull/14#discussion_r497978393) in the notebook as well.

Reply:
Yeah. Will do after some time.

Reply:
The links you have provided in <td> tags are not working. Please check whether these are correct.

Reply:
That's right, they would work once this PR is merged

Reply:
Okay